packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -9,64 +9,117 @@
-                  Learn more in our blog post.
-                </ExternalLink>
+                Cypress now ensures each test runs in a clean browser context by default. We now clear the page, <code>cookies</code>, <code>localStorage</code>, and <code>sessionStorage</code> before each test to guide developers towards writing independent tests from the start.
               </p>
               <p>
-                Existing Component Testing users, see the
-                <ExternalLink
-                  href="https://on.cypress.io/changelog#11-0-0"
-                >
-                  <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
-                  changelog</ExternalLink>
-                to learn about breaking changes.
+                If your existing tests relied on a previous test to run successfully, you might need to make some modifications to your test suite. See the
+
+                <ExternalLink href="https://on.cypress.io/migration-guide#Migrating-to-Cypress-12-0">
+                  Migration Guide
+                </ExternalLink>
+
+                for more details on what you can expect.
               </p>
+
               <h2 class="font-bold mt-24px mb-16px text-18px text-jade-1000">
-                Faster Startup Time
+                We Now Store The Browser Context, So You Don’t Have To
               </h2>
 
               <p>
-                We have also massively improved our startup performance by shipping a snapshot of our binary instead of the source files. Results will vary based on your situation, but we saw up to 84% faster startup times!
+                The <ExternalLink href="https://on.cypress.io/session">
+                  <code>cy.session()</code>
+                </ExternalLink> command complements test isolation by providing a way to save and share browser contexts between tests and specs in a single run on the same machine.
               </p>
+
+              <h2 class="font-bold mt-24px mb-16px text-18px text-jade-1000">
+                Detaching Ourselves From Detached Dom Errors
+              </h2>
+
               <p>
-                For a complete list of updates in v11, please review our <ExternalLink
-                  href="https://on.cypress.io/changelog#11-0-0"
-                >
-                  <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
-                  changelog</ExternalLink>.
+                We have made enhancements to how Cypress manages DOM element resolution to reduce the likelihood of hitting the dreaded detached DOM errors due to maintaining stale DOM references we have all experienced from time-to-time. We've updated our
```

Comment:
Just a bit of a run on sentence here. 

```suggestion
                We have made enhancements to how Cypress manages DOM element resolution to reduce the likelihood of hitting the dreaded detached DOM errors due to maintaining stale DOM references. We've updated our
```